packages/expo/package.json

```diff
@@ -23,6 +23,7 @@
     "expo": "^40.0.0",
     "expo-constants": "~9.3.3",
     "expo-facebook": "~9.1.0",
+    "expo-localization": "~9.1.0",
```

Comment:
I recommend you add the i18n package as dependency here as well. We're gonna update all the package.json files adding the respectives dependencies in order to keep the consistence among them.

Reply:
#82